helm/trento-ai-companion/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: trento-ai-companion
```

Comment:
Since the helm chart contains the `mcp-server-trento` and the `mcpo` proxy I just came up with this name, but happy to rename it to an alternative name. cc @stefanotorresi